controllers/destination_controller.go

```diff
@@ -77,17 +77,17 @@ func (r *DestinationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	logger = logger.WithValues("path", path)
 
 	if err := r.createDependenciesPathWithExample(writer); err != nil {
-		logger.Error(err, "unable to write dependencies to state store")
+		logger.Info("unable to write dependencies to state store", "error", err)
 		return defaultRequeue, nil
 	}
 
 	if err := r.createResourcePathWithExample(writer); err != nil {
-		logger.Error(err, "unable to write dependencies to state store")
+		logger.Info("unable to write resources to state store", "error", err)
 		return defaultRequeue, nil
 	}
 
 	if err := r.Scheduler.ReconcileDestination(); err != nil {
-		logger.Error(err, "unable to schedule destination resources")
+		logger.Info("unable to schedule destination resources", "error", err)
 		return defaultRequeue, nil
```

Comment:
🤔 it is an error, but gracefully handeled. I don't know if logging info or error is better in this scenario 🤷

Reply:
After offline conversations this is going to stay an open question. We need to figure out our logging levels for coding standards and deciding how to indicate reconciliation errors that will be retried will fall into that conversation.